centos/nap/entrypoint.sh

```diff
@@ -48,9 +48,6 @@ test -n "${ENV_CONTROLLER_API_KEY}" && \
 test -n "${ENV_CONTROLLER_INSTANCE_NAME}" && \
     instance_name=${ENV_CONTROLLER_INSTANCE_NAME}
 
-test -n "${ENV_CONTROLLER_URL}" && \
```

Comment:
the controller url should remain a runtime setting as well as a build time setting.
It is an option for the customer regarding both how they want to build and deploy the image.
We don't won't to force one image per controller instance - which is what we impose by removing the runtime option.

Reply:
In the entrypoint.sh script, ENV_CONTROLLER_URL was used to set value of `api_url` variable in agent.conf.
But it was before `ENV_CONTROLLER_URL` format changed from:
`https://<fqdn>:8443/1.4`
to
https://<fqdn>/install/controller-agent

From what we know from agent squad, new format is not valid for `api_url` field and `api_url` field filled automatically at the time of installation ( @1996sajal could you confirm it? As you are the one who talked with agent squad).
So `controller_url` != `api_url` after last changes.
That's why we removed this assignment from our script.

If we want user to be able to change `api_url` to different one, we may need to introduce new variable?

Reply:
I added story linked to this issue:
https://nginxsoftware.atlassian.net/browse/IND-20216